src/client/dfs/io.c

```diff
@@ -85,11 +100,13 @@ dfs_read_int(dfs_t *dfs, dfs_obj_t *obj, daos_off_t off, dfs_iod_t *iod, d_sg_li
 	if (rc)
 		D_GOTO(err_params, rc);
 
+	DFS_OP_STAT_INCR(dfs, DOS_READ);
```

Comment:
is this counting the number of read operations?
should you then move it the completion cb too after the op succeeds?
for other operations like mkdir, etc you only increment after the op succeeds not if the op fails.

Reply:
Oops. I must have screwed up a rebase. I had moved this into the callback. Sigh. I'll go back through and check it all again more carefully.